apps/music/js/metadata/flac.js

```diff
@@ -92,20 +112,21 @@ var FLACMetadata = (function() {
 
       // Get the contents of this block, plus enough to read the next block's
       // header if necessary.
-      blobview.getMore(blobview.viewOffset + blobview.index, length + 4,
-                       function(more, err) {
-        if (err) {
-          reject(err);
-          return;
-        }
+      blobview.getMore(
+        blobview.viewOffset + blobview.index, length + 4,
+        function(more, err) {
+          if (err) {
+            reject(err);
+            return;
+          }
 
-        resolve({
-          last: last,
-          block_type: block_type,
-          length: length,
-          view: more
+          resolve({
+            last: last,
+            block_type: block_type,
+            length: length,
+            view: more
+          });
```

Comment:
These are just whitespace changes, right? It might be worth reverting them to keep the diff smaller, but I don't feel strongly about it one way or the other.